### PR TITLE
added new builder for sklearn style mlp

### PR DESCRIPTION
**Pull request by john-waczak**

### CODE REVIEW

**Review by ablaom — APPROVED** [review]
Many thanks! Looks good to me.

File: src/builders.jl [hunk 1]
```diff
@@ -62,6 +62,42 @@ function build(builder::Short, rng, n, m)
...
+    init=Flux.glorot_uniform(rng)
+    hidden = []
+    for i ∈ 2:length(mlp.hidden)
+        push!(hidden, Flux.Dense(mlp.hidden[i-1], mlp.hidden[i], mlp.σ, init=init))
+    end
+
+    return Flux.Chain(
+        Flux.Dense(n_in, mlp.hidden[1], mlp.σ, init=init),
+        hidden...,
+        Flux.Dense(mlp.hidden[end], n_out, init=init)
```
Comment (ToucheSir) [inline src/builders.jl:90]: The first and last layers can probably be included in the definition of/`push!`ed into `hidden` directly. Then the splat can cover everything.

File: src/builders.jl [hunk 2]
```diff
@@ -62,6 +62,42 @@ function build(builder::Short, rng, n, m)
...
+The each layer is initialized using `Flux.glorot_uniform(rng)`. If
+`rng` is an integer, it is instead used as the seed for a
+`MersenneTwister`.
+
+"""
+mutable struct MLP{N} <: MLJFlux.Builder
+    hidden::NTuple{N, Int}  # count first and last layer
+    σ
+end
+MLP(; hidden=(10,), σ=Flux.sigmoid) = MLP(hidden, σ)
```
Comment (ToucheSir) [inline src/builders.jl:79]: SkLearn's `MLP{Classifier|Regressor}` use 100 hidden units and ReLU instead of sigmoid. Was there a particular reason for the choice of these hyperparams?

**Review by ToucheSir — COMMENTED** [review]
Couple of drive-by comments from me (which need not hold up this PR)

File: src/builders.jl (hunk 1 above)
Comment (john-waczak) [reply src/builders.jl:90]: That makes sense. In my head I was keeping the input/output layers separate from the hidden layers.

File: src/builders.jl (hunk 2 above)
Comment (john-waczak) [reply src/builders.jl:79]: No particular reason. I think I grabbed sigmoid just because I started by looking at the implementation of `Short`. ReLU will probably give a faster model for predictions.

**Comment by ablaom** [comment]
@ToucheSir Thanks for chiming in here.

@john-waczak Would you mind submitting an updated PR?
